Graded/Restrictions.agda

```diff
@@ -561,12 +570,12 @@ opaque
 
   -- If grade equality is decidable and the modality comes with a
   -- dedicated nr function, then UD.Assumptions holds for
-  -- no-usage-restrictions b false.
+  -- no-usage-restrictions Nr b false.
```

Comment:
The code in the comment is underspecified (note that `Nr` takes an instance argument).